MAUI/PDF-Viewer/Coordinate-Conversion.md

```diff
@@ -0,0 +1,119 @@
+
+![Coordinate conversion in .NET MAUI PDF Viewer](Images\Coordinate-Conversion\Client-Coordinates.jpg)
+
+### Get page number from client point
+
+You can obtain the page number of the client point using the `GetPageNumberFromClientPoint(Point clientPoint)` method by passing the client point as input parameters. The following code sample explains how to get the page number from the client point.
+
+{% tabs %}
+{% highlight c# %}
+//Subscribe the Tapped event of PDF Viewer
+pdfViewerControl.Tapped += PdfViewerControl_Tapped; 
+ 
+// Handle the tapped event.
+private void PdfViewerControl_Tapped(object sender, GestureEventArgs e)
+{
+    //Retrieve the tapped client area position.
+    Point clientPoint = e.Position;
+    //Retrieve the page number that corresponds to the client point.
+    int pageNumber = pdfViewerControl.GetPageNumberFromClientPoint(clientPoint);
+}
+{% endhighlight %}
+{% endtabs %}
+
+## PDF page coordinates
+
+PDF page coordinates are represented in terms of device-independent coordinate system called user space, which is independent of the output device that will be used for printing or display. The user space coordinate system is initialized to a default state for each page of a PDF document. The length of a unit is 1/72 inch, which is approximately the same as a unit of point(pt). As an example, the dimensions of a letter-sized paper in PDF page coordinates are shown in the diagram below.
+
+![Coordinate conversion in .NET MAUI PDF Viewer](Images\Coordinate-Conversion\Page-Coordinates.jpg)
+
+## Convert PDF Viewer’s coordinates to PDF page coordinates
+
+You can obtain the PDF page coordinates using the `ConvertClientPointToPagePoint(clientPoint, pageNumber)` method by passing the client rectangle point and page number as input parameters. The following code sample explains how to convert a tapped client area position to a page point.
+
+{% tabs %}
+{% highlight c# %}
+//Subscribe the Tapped event of PDF Viewer
+pdfViewerControl.Tapped += PdfViewerControl_Tapped; 
+ 
+// Handle the tapped event.
+private void PdfViewerControl_Tapped(object sender, GestureEventArgs e)
+{
+    //Retrieve the tapped client area position.
+    Point clientPoint = e.Position;
+    //Retrieve the page number that corresponds to the client point.
+    int pageNumber = pdfViewerControl.GetPageNumberFromClientPoint(clientPoint)
+    //Retrieve the page point.
+    Point pagePoint = pdfViewerControl.ConvertClientPointToPagePoint(clientPoint, pageNumber);
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Convert PDF page coordinates to PDF Viewer’s coordinates
+
+You can obtain the PDF page coordinates using the `ConvertPagePointToClientPoint(pagePoint, pageNumber)` method by passing the page point and page number as input parameters. The following code sample explains how to convert a square annotation’s position in the page coordinates to a client point.
```

Comment:
Include the following things.
1. Mention the square annotation, position is represented in the PDF page coordinates. 
2. As you used **0th** index for square annotation, say the document contain square annotation as an example,
3. Similarly you can modify based on your use cases, Example https://help.syncfusion.com/maui/pdf-viewer/add-remove-modify-annotations#add-annotations-programmatically

Reply:
addressed